pkg/components/util/types.go

```diff
@@ -44,7 +44,7 @@ type Toleration struct {
 	Effect            string `hcl:"effect,optional" json:"effect,omitempty"`
 	Operator          string `hcl:"operator,optional" json:"operator,omitempty"`
 	Value             string `hcl:"value,optional" json:"value,omitempty"`
-	TolerationSeconds string `hcl:"toleration_seconds,optional" json:"toleration_seconds,omitempty"`
```

Comment:
Again, rather than explaining what is done in commit message (which is easy to deduct), please explain why it's done (harder part).